src/Common.am

```diff
@@ -73,3 +73,11 @@ COMPAT_LIB = $(top_builddir)/compat/libcompatsquid.la $(LIBPROFILER)
 
 ## Some helpers are written in Perl and need the local shell defined properly
 subst_perlshell = sed -e 's,[@]PERL[@],$(PERL),g' <$(srcdir)/$@.pl.in >$@ || ($(RM) -f $@ ; exit 1)
+
+EXTENSIONS=.hdrtest
+.h.hdrtest:
+	@echo "testing header $<"
+	@sed 's%[@]HEADER[@]%${<}%' $(top_srcdir)/test-suite/testHeader.cc.in | \
```

Comment:
FYI;  If you remove the `@` from `@sed` the build log should get a printout of the test commands run so we can manually check what test logic is happening when the weirdly now-broken headers fail to pass.

Reply:
I know, I put there on purpose.
The question is not why the tests are failing, but why they used to succeed.

Most of the tests have a pattern in common: RefCount<C> requires a full definition of C to invoke its destructor, a forward declaration is not enough. In many cases the naked header doesn't have that

Reply:
That is hard to tell when the command is hidden. All I can see in the log is that the latest headers are failing because they are trying to be built in the `srcdir` instead of `builddir` - the working ones were writing to the `builddir` along with every other generated file. It may be as simple as fixing that so the relative paths work properly.